### PR TITLE
improve docs

### PR DESCRIPTION
**Pull request by mribeirodantas**
- Add -f, -fields to log CLI help
- Remove whitespace
- Fix typo in aws.rst

### CODE REVIEW

**Comment by bentsherman** [comment]
@mribeirodantas please hold off on making broad docs changes like this until we have merged the markdown PR. Every time the RST docs are updated I have to resolve a merge conflict.

**Comment by mribeirodantas** [comment]
Ok! No problem 😄

**Comment by pditommaso** [comment]
Sorry Ben, let's move this ahead. I've also added some stuff to the RST docs. Let's plan the markdown version post 22.10.0 nextflow release.

**Review by pditommaso — APPROVED** [review]

**Pull request merged**
